README.md

```diff
@@ -1,8 +1,12 @@
 # Lunch-Hackers
```

Comment:
I agree. Maybe we should nuke this and start a new repo, with all new structure!